flight-website/app/templates/application.hbs

```diff
@@ -1,5 +1,8 @@
 {{page-title "Flight icons"}}
 <DsHeader />
+<div class="ds-beta-banner">
+  <strong>Notice</strong>: there is a new "beta" version of this website:
+  <a href="https://hds-website-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">do you want to try it?</a></div>
```

Comment:
<img width="575" alt="Screen Shot 2023-01-04 at 2 18 23 PM" src="https://user-images.githubusercontent.com/2200899/210661030-c8cf8d7d-bfca-4547-83c7-6efccfa8337a.png">
This link doesn't have an underline, not immediately clear that its a link. Must be overridden somewhere else

Reply:
Fixed. Not sure how I missed it 🤦‍♂️